index.html

```diff
@@ -12,15 +11,22 @@
     <title>My Portfolio</title>
 </head>
 <body>
-        <header class="main-header"> 
-            <a class="logo" href="#"><img src="images/My Logo (1).png" alt="home page logo"></a>
-            <nav id="nav-container">
-                    <a class="desk" href="#" >Project</a>
-                    <a class="desk" href="#" >About</a> 
-                    <a class="desk" href="#" >Contact</a>
-            <a class="menu" href="#" ><img src="icon/menu.png"alt="menu"></a>    
-        </nav>
-    </header>
+        <header>
+           <a class="logo" href="#"><img src="images/My Logo (1).png" alt="home page logo"></a>
+            <div class="menu-toggle">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+            <nav>
+            <button class="close-button">&times;</button>
+              <ul class="links">
+                <li><a href="#portfolios">Portfolio</a></li>
+                <li><a href="#about-section">About</a></li>
+                <li><a href="#contacts-form">Contact</a></li>
+              </ul>
+            </nav>
+          </header>
```

Comment:
- [ ] You did a great job 😃  at using the font family for the links, but as stated by the previous reviewer kindly 💯 try to stick to the Figma Design at the background color of the mobile menu, so let's try to use the same background color from the Figma to make it nice like the Figma 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/232997396-5e0bcdf8-4ae1-4f42-a3be-52ce4a7b2e19.png)   |  ![image](https://user-images.githubusercontent.com/83360636/232997249-6dea1516-15a6-4ecb-a092-76736dc48381.png)|